content/03.methods.md

```diff
@@ -11,10 +11,50 @@
 2. Describe metadata statistics on PubMed central (number of articles, how many articles were processed
 
 ### Comparing Corpora
-1. Spacy to process text via - Lemmatization, removal of stop words
-2. Describe counting frequencies of each lemma
-3. Describe using chi-square test
-4. Describe how to calculate the likelihood and log odds ratio
+The first step in linguistic analysis is to preprocess the text for ease of computation.
+This step consists of removing non-informative words (i.e., stop words) as well as converting words into their root form (i.e., lemmatization).
+Throughout our analysis we encountered symbols that aren't considered words (e.g., $\pm$), so we will refer to words and symbols as tokens to avoid confusion.
+We used spaCy [@raw:honnibal2017spacy] to perform the above preprocessing tasks on the bioRxiv and PubMed Central corpora.
+
+Following the cleaning process, we calculated the frequency of each token between both corpora.
+Due to the large influx of unique tokens, we subsampled our analysis to use the union of the top 100 most frequent tokens between both corpora.
+For each token we generated a contingency table as described in [@doi:10.1075/ijcl.6.1.05kil] (Table {@tbl:term-freq-contingency-table}).
+
+| | corpus X | corpus Y | |
+|---|---|---|---|
+| w | a | b | a+b |
+| $\neg$w | c | d | c+d |
+| | a+c | b+d | a+b+c+d = Total |
+
+Table: A contingency table for a given token that may appear in two corpora that are being compared.
+Given a query token $w$, the first row depicts the frequency of $w$ appearing in corpus X (a) and corpus Y (b).
+The second row depicts the frequency of all other tokens appearing in corpus X (c) and corpus Y (d).
+The total number of tokens can be found by adding (a,b,c,d) together as depicted in the bottom right.
+{#tbl:term-freq-contingency-table}
+
+Once the contingency table has been created, we calculated the odds ratio  for a given token.
+The odds ratio is defined as the fraction of the probability of a token $w$ appearing in corpus X divided by the probability of a token $w$ appearing in corpus Y.

```

Comment:
This is also more introductory than I think you want in this paper. Similar to above I would recommend removing this definition, but one day for an introductory review this would be great to have :)